content/release-notes/1.225.md

```diff
@@ -0,0 +1,49 @@
+
+**Note:** This change affects customers who use container images from the `instana` namespace in the public Docker Hub registry. Images that are made available by using Instana's own registry at `containers.instana.io` are not affected by this change.
+
+Make sure to update all references to the public Docker Hub images with IBM Cloud Container Registry image locations in Dockerfiles, build scripts, or other files. The following table lists the images with their new locations:
+
+| Docker Hub Image                   | IBM Cloud Container Registry Image           |
+| ---------------------------------- | -------------------------------------------- |
+| `instana/agent`                    | `icr.io/instana/agent`                       |
+| `instana/aws-fargate-nodejs`       | `icr.io/instana/aws-fargate-nodejs`          |
+| `instana/aws-lambda-nodejs`        | `icr.io/instana/aws-lambda-nodejs`           |
+| `instana/google-cloud-run-nodejs`  | `icr.io/instana/google-cloud-run-nodejs`     |
+| `instana/instana-agent-operator`   | `icr.io/instana/instana-agent-operator`      |
+| `instana/instrumentation`          | `icr.io/instana/instrumentation`             |
+| `instana/leader-elector`           | `icr.io/instana/leader-elector`              |
+
+For more information, see [Changes to the Instana container image repositories](https://www.ibm.com/docs/en/obi/current?topic=instana-host-agent#changes-to-the-instana-container-image-repositories).
+
+## Features
+
+* Website Smart Alerts on custom events
+  - Instana now supports alerting on **Custom Events** of your monitored website. This helps to notify when a specific event occurs.
+    For more information, see [Website Smart Alerts docs](https://www.ibm.com/docs/en/obi/current?topic=websites-smart-alerts#custom-events).
+
+* Camunda tracing
+  - Instana now supports Camunda tracing of BPMN activities.
+    - By using this feature, you can gain visibility into Camunda BPMN activities, and automatically link the activity-related sub calls for full context and infrastructure insights.
+
+## Improvements
+
+* OracleDB Sensor now collects service names for each database instance.
+
+* IBM i OS Sensor:
+  - Validated the service availability for all the metrics.
+
+* Added a new aggregation **per second**. The per second aggregation is available for the `calls` and `erroneousCalls` metrics in **Unbounded Analytics** page.
+
+* Application Perspectives: Replaced `Calls` and `Erroneous Calls` widgets with `Calls Per Second` and `Erroneous Call Rate` in the **Summary** tab.
+
```

Comment:
```suggestion

* Smart Alerts: Improved usability of configuration form: it scrolls automatically to invalid field input

```

Reply:
As mentioned [here](https://github.com/instana/ui-notifications/pull/38#discussion_r871180496). Please update the respective section in the docs repository first. 😇 

Thanks.